demos/bubble_shear.py

```diff
@@ -0,0 +1,237 @@
+# Advection of a tracer bubble in a non-uniform shear flow
+# ===================================================
+
+# In this demo, we consider the advection of a scalar concentration field, similarly to
+# our previous solid body rotation demo. However, we now consider a background velocity
+# field that is not uniform in time. We further prescribe homogeneous Dirichlet boundary
+# conditions for the concentration, so in this demo we solve
+#
+# .. math::
+#   \begin{array}{rl}
+#       \frac{\partial c}{\partial t} + \nabla\cdot(\mathbf{u}c)=0& \text{in}\:\Omega\\
+#       c=0 & \text{on}\:\partial\Omega,\\
+#       c=c_0(x,y) & \text{at}\:t=0,
+#   \end{array},
+#
+# where :math:`c=c(x,y,t)` is the sought tracer concentration, :math:`\mathbf{u}=\mathbf{u}(x,y,t)` is the
```

Comment:
Please keep lines to 89 characters, except in the `.. math::` blocks.

Reply:
Fixed in 262c78a51a99714a4794b55259edf6fc5aea3567 and I'll keep an eye on it if I make further changes! Thanks :)